spec/features/logging_out_spec.rb

```diff
@@ -0,0 +1,13 @@
+feature 'logging a user out' do
```

Comment:
can add a log in feature test as well, using User.register to register user beforehand